src/2048-expert/index.js

```diff
@@ -269,23 +274,85 @@ function newGame() {
     clearBoard()
     generateNewTile()
     generateNewTile()
+    addColours()
 }
 
-// Reset Current Score and reset all values to 0
+
+// function clear Board
 function clearBoard() {
-    cells.forEach(cell => cell.innerHTML = "")
+    cells.forEach(cell => {
+    cell.innerHTML = ""
+    cell.setAttribute('data-colour', 0)
+    })
     currentScore = 0
     currentScoreDisplay.innerHTML = 0
 }
 
+// Add colour 
+function addColours() {
+    for (let i = 0; i < totalCell; i++) {
+        if (cells[i].innerHTML == 0) {
```

Comment:
Are you able to refactor this into a switch statement e.g.

const number = cells[i].innerHTML

```
switch(number) {
    case 0:
    .....
}
```

This will help reduce some repetition.